src/include/iscsi-lio-server/UI_dialogs.rb

```diff
@@ -906,7 +906,7 @@ def GetNetConfig
     ip_list = []
     re_ipv4 = Regexp.new(/[\d+\.]+\//)
     re_ipv6 = Regexp.new(/[\w+\:]+\//)
-    ret = Yast::Execute.locally('ip', 'a', stdout: :capture)
+    ret = Yast::Execute.locally('/sbin/ip', 'a', stdout: :capture)
```

Comment:
@mchf don't we have some code in yast2-network for doing this?

Reply:
no, yast2-network doesn't have a method for listing UP interfaces and its IPs